src/Microsoft.AspNet.Mvc.Core/ActionResults/FileContentResult.cs

```diff
@@ -19,13 +20,14 @@ public class FileContentResult : FileResult
 
         /// <summary>
         /// Creates a new <see cref="FileContentResult"/> instance with
-        /// the provided <paramref name="fileContents"/>.
+        /// the provided <paramref name="fileContents"/> and the
+        /// provided <paramref name="contentType"/>.
         /// </summary>
         /// <param name="fileContents">The bytes that represent the file contents.</param>
-        public FileContentResult([NotNull] byte[] fileContents)
-            : base(contentType: null)
```

Comment:
This seems to be like a compatiblity bug fix - should have been dealt with as a separate PR. (Its fine to leave this here for this one, but something to keep in mind in general ).